backend/src/config/color.rs

```diff
@@ -11,16 +11,25 @@ pub(crate) struct Color {
 }
 
 impl Color {
-    /// Returns either "black" or "white", depending on which of those has a
-    /// higher contrast to `self`.
+    /// Calculates relative luminance and contrast ratio of `self` to white,
+    /// based on WCAG2.1 Technique G18 (https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html).
+    /// Returns either "black" or "white", depending on contrast ratio.
```

Comment:
Rust documentation (with `///`) is rendered as Markdown. Not that we usually do that in Tobira, but if you really want you can run `cargo doc --open`.

I would probably write it like this:

```
    /// Returns either "black" or "white", depending on which has the higher 
    /// contrast to `self`. Contrast calculation based on [WCAG2.1 Technique G18][1].
    ///
    /// [1]: https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html
```